python/hsfs/feature_group.py

```diff
@@ -154,9 +160,26 @@ def insert(self, features, overwrite=False, storage=None, write_options={}):
             write_options,
         )
 
+        if self.statistics_config.enabled:
+            self._statistics_engine.compute_statistics(self, self.read(storage))
+
     def delete(self):
         self._feature_group_engine.delete(self)
 
+    def update(self):
+        self._feature_group_engine.update(self)
+        return self
+
+    def compute_statistics(self, storage=None):
+        """Recompute the statistics for the feature group and save them to the
+        feature store.
+
+        :param storage: storage to read data from, defaults to None
+        :type storage: str, optional
+        """
+        if self.statistics_config.enabled:
+            self._statistics_engine.compute_statistics(self, self.read(storage))
```

Comment:
`storage` here can be `ALL` meaning that the insert will be written both in the online and offline feature store.
Read will throw an exception if `ALL` is set, so the user will have an exception during the insert.

Also I don't think it's wise to let the user compute the statistics over the online feature store. It will trigger a scan which is not great for NDB.

Reply:
okay if storage is `ALL` I can just compute it on the offline FS, but do we really want to disable statistics completely for the online feature store?

Reply:
Removed the possibility to specify a storage to compute statistics from and changed it so it issues a warning if the default storage is `online`. And read from offline if default storage is `ALL`.